### DISCUSSION TITLE
Is there a way to configure flex-search to work for markdown tables?

### DISCUSSION THREAD

**Discussion by nktnet1** [fuma-nama/fumadocs#770] [Q&A] [answered]
I have the following in my NextJS project as per the "Advanced Search" page:
https://fumadocs.vercel.app/docs/headless/search/flexsearch#advanced-search

```ts
import { getPages } from '@/app/source';
import { createSearchAPI } from 'fumadocs-core/search/server';

export const { GET } = createSearchAPI('advanced', {
  indexes: getPages().map((page) => ({
    title: page.data.title,
    structuredData: page.data.exports.structuredData,
    id: page.url,
    url: page.url,
  })),
});
```

which works great, although it isn't able to pick up text from markdown tables, e.g. searching for "Alice" in

```md
| Name     | Age | Occupation  |
|----------|-----|-------------|
| Alice    | 25  | Engineer    |
| Bob      | 30  | Designer    |
| Charlie  | 35  | Teacher     |
```

Are there any workarounds to make this possible without losing other functionalities?

**Comment by fuma-nama** [comment, marked as answer]
Just released a fix, now it should index tables correctly

**Reply by nktnet1** [reply]
Thank you so much!!

**Reply by nktnet1** [reply]
Hi @fuma-nama,

I have a similar issue, this time for the description metadata when using flex-search. Do you think this is related to what happened here?

For example

```markdown
---
title: This can be found
description: But not this
---
```

**Reply by fuma-nama** [reply]
You probably forgot to pass description to your search API, it's optional because in the past, description was only a SEO property and isn't actually rendered. You can add a description property to search indexes (you can find it at route.ts)

**Reply by nktnet1** [reply]
That was it - thank you!
